sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -38,8 +57,80 @@ static Statsbeat()
 
         private static Measurement<int> GetAttachStatsBeat()
         {
-            // TODO: Add additional properties required for statbeat
-            return new(1, new("cikey", Customer_Ikey), new("language", "dotnet"));
+            if (s_resourceProvider == null)
+            {
+                SetResourceProviderDetails();
+            }
+
+            // TODO: Add os to the list
+            return
```

Comment:
nit:
```suggestion
            return
                new Measurement<int>(1,
                     new("rp", s_resourceProvider),
                      new("rpId", s_resourceProviderId),
                      new("attach", "sdk"),
                      new("cikey", Customer_Ikey),
                      new("runtimeVersion", s_runtimeVersion),
                      new("language", "dotnet"),
                      new("version", s_sdkVersion));
```

Reply:
still alignment makes it difficult to read, dimensions are parameter a little indent will help read.

Reply:
is this better now?

Reply:
Yes